src/librustc/ty/layout.rs

```diff
@@ -1700,10 +1718,9 @@ impl<'a, 'tcx, C> TyLayoutMethods<'tcx, C> for Ty<'tcx>
         let tcx = cx.tcx();
         let discr_layout = |discr: &Scalar| -> C::TyLayout {
             let layout = LayoutDetails::scalar(cx, discr.clone());
-            MaybeResult::from_ok(TyLayout {
-                details: tcx.intern_layout(layout),
-                ty: discr.value.to_ty(tcx)
-            })
+            MaybeResult::from(Ok(
+                TyLayout {details: tcx.intern_layout(layout),ty: discr.value.to_ty(tcx)}
```

Comment:
Please keep the formatting from before (you can use `Tools -> Rustfmt` on the playground if you're unusure how to format something - e.g. https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=7e17e5eef9cf0076872fec35670b86ea)